modules/managed-identity/user-assigned-identity/README.md

```diff
@@ -0,0 +1,68 @@
+# User Assigned Identities
+
+This module deploys Microsoft.ManagedIdentity User Assigned Identities and optionally available children or extensions
+
+## Parameters
+
+| Name              | Type     | Required | Description                                                                                                                                                                                                                                                                                                                                                                                                    |
+| :---------------- | :------: | :------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `name`            | `string` | Yes      | Required. Name of the User Assigned Identity.                                                                                                                                                                                                                                                                                                                                                                  |
+| `location`        | `string` | No       | Optional. Location for all resources.                                                                                                                                                                                                                                                                                                                                                                          |
+| `lock`            | `string` | No       | Optional. Specify the type of lock.                                                                                                                                                                                                                                                                                                                                                                            |
+| `roleAssignments` | `array`  | No       | Optional. Array of role assignment objects that contain the 'roleDefinitionIdOrName' and 'principalId' to define RBAC role assignments on this resource. In the roleDefinitionIdOrName attribute, you can provide either the display name of the role definition, or its fully qualified ID in the following format: '/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11' |
```

Comment:
it probably makes sense to provide a code sample of the shape of the object for object params...  deciphering in a single line is harder than it should be...  I think we have:
```bicep
{
  principalId: 'foo'
  roleDefinitionIdOrName: 'bar'
}
```
but I also thought that principalId was (or could be) an array...